File: Lesson 1 - C/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    /*  for (int i = 0; i < 4; i++)
     {
         printf("?");
     }
     printf("\n"); */

    /*  for (int i = 1; i < 10; i++)
     {
         printf("#");
         if (i % 3 == 0){
             printf("\n");
         }
     } */
    /*  int n = get_int("Size: ");

     while (n < 1)
     {
         n = get_int("Please enter a positive number: ");
     } */

    // Prompt user for positive integer
    int n;
    do
    {
        n = get_int("Size: ");
    } while (n < 1);

    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Lesson 6 - Python/sentimental-mario-more/mario.py
# TODO
import cs50


heigth = cs50.get_int("Input an integer: ")
while heigth < 0 or heigth >= 8:
    heigth = cs50.get_int("Input an integer: ")
    
    
for i in range(heigth):
    for j in range(heigth):
        j = heigth - i - 1
        print(" ")
    for k in range(heigth):
        k = i + 1
        print("#")
    print("  ")
    for l in range(heigth):
        l = i + 1
        print("#")
